package.json

```diff
@@ -14,6 +14,7 @@
     "@react-navigation/stack": "^5.14.2",
     "axios": "^0.21.1",
     "expo": "~40.0.0",
+    "expo-app-loading": "^1.0.1",
```

Comment:
try to install lesser dependencies as it increases app size.

Reply:
we already have built-in loaders - `Activity Indicator`. no need of this library